firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionHeaderView.swift

```diff
@@ -3,93 +3,138 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import Foundation
-import UIKit
 import Common
-import Shared
+import SiteImageView
 import ComponentLibrary
 
-class TrackingProtectionHeaderView: UIView {
-    let siteTitleLabel: UILabel = .build { label in
-        label.adjustsFontForContentSizeCategory = true
-        label.textAlignment = .center
-        label.font = FXFontStyles.Regular.headline.scaledFont()
-        label.numberOfLines = 2
-        label.accessibilityTraits.insert(.header)
+final class TrackingProtectionHeaderView: UIView, ThemeApplicable {
+    private struct UX {
+        static let headerLinesLimit: Int = 2
+        static let siteDomainLabelsVerticalSpacing: CGFloat = 12
+        static let faviconImageSize: CGFloat = 40
+        static let closeButtonSize: CGFloat = 30
+        static let horizontalMargin: CGFloat = 16
     }
 
-    private var closeButton: CloseButton = .build { button in
-        button.layer.cornerRadius = 0.5 * TPMenuUX.UX.closeButtonSize
+    var closeButtonCallback: (() -> Void)?
+
+    private var faviconHeightConstraint: NSLayoutConstraint?
+    private var faviconWidthConstraint: NSLayoutConstraint?
+
+    private lazy var headerLabelsContainer: UIStackView = .build { stack in
+        stack.backgroundColor = .clear
+        stack.alignment = .leading
+        stack.axis = .vertical
+        stack.spacing = TPMenuUX.UX.headerLabelDistance
```

Comment:
Another sneaky `TPMenuUX`

Reply:
This is used in two files only... I can duplicate it but for sure will not be the best option, from my perspective.

Reply:
You have more context, so you can decide. It sounds like it's best to leave it, like you say. 👍